pkg/cmd/workspace/start.go

```diff
@@ -108,6 +117,21 @@ var StartCmd = &cobra.Command{
 			return err
 		}
 
+		workspace, err := apiclient_util.GetWorkspace(workspaceIdOrName, false)
+		if err != nil {
+			return err
+		}
+
+		if startProjectFlag != "" {
+			projectNames = append(projectNames, startProjectFlag)
+		} else {
+			projectNames = util.ArrayMap(workspace.Projects, func(p apiclient.Project) string {
+				return p.Name
+			})
+		}
+
+		apiclient_util.ReadWorkspaceLogs(ctx, activeProfile, workspace.Id, projectNames, false, true, timeNow)
```

Comment:
While this does work and the logs get read correctly, this is not the desired outcome.

The logs should be read while the workspace/project is starting. This means that this function should be called as a goroutine above the `StartProject` function.

Additionally, since the spinner will bother the logs, we can remove the spinner completely for the start command and only display the logs + the final message.

Lastly, from what I can see, we can move the log read call into `StartWorkspace` so we don't have to repeat it every time.

Reply:
Gotcha! But if we move the log read call to the StartWorkspace that will mean when we use `dtn code | restart` logs for that will also be fetched. Is it alright?

Reply:
Done.
<img width="718" alt="Screenshot 2024-10-09 at 6 23 31 PM" src="https://github.com/user-attachments/assets/e40d4637-8bc3-4f20-b82a-53e740967309">